components/HamburgerMenu.vue

```diff
@@ -102,6 +127,12 @@ function closeMenu() {
     isMenuOpen.value = false
 }
 
+function logout() {
+    authStore.logout()
+    closeMenu()
+}
+
+
 //this is a custom directive to close the menu when clicking outside of the menu
 const vCloseOnOutsideClick = {
     mounted: (el: any) => {
```

Comment:
```suggestion
    mounted: (el: HTMLElement) => {
```

No anyscript!

Reply:
oh, this is unrelated to auth, but I'm gonna fix it!

Reply:
nevermind, these types are annoying to fix, so let's move them to a separate PR to finish cleaning up types